client/src/components/Inbox/MessagesContainers.js

```diff
@@ -97,6 +98,7 @@ export const MessagesContainer = styled.div`
       }
       return "5.1rem;";
     }}
+    top: 11rem;
```

Comment:
I've tried to remove the [RecipientHeader.js z-index of 2](https://github.com/FightPandemics/FightPandemics/blob/1ee3751f8eb03df1d3f4625997b03261607577fd/client/src/components/Inbox/RecipientHeader.js#L31), but that doesn't align RecipientHeader and MessagesContainer side by side vertically, so trying to add a top margin on MessageContainer, which could be wrong way to fix it.